assets/scss/_styles_project.scss

```diff
@@ -52,39 +52,6 @@
   background: $primary !important;
 }
 
-// Icons
-#main_navbar {
-  li i {
-  padding-right: 0.5em;
-
-      &:before {
-          display: inline-block;
-          text-align: center;
-          min-width: 1em;
-      }
-  }
-.alert {
```

Comment:
Btw, this nested definition / override for `.alert` under `#main_navbar`, doesn't make any sense to me -- I can't see a case where a standard Bootstrap alert would be used inside the navbar. (Interestingly, the website doesn't have any alerts at all.)